client/src/pages/Signup.jsx

```diff
@@ -1,13 +1,286 @@
 import React from 'react';
-import Link from '@material-ui/core/Link';
+import Button from '@material-ui/core/Button';
+import CssBaseline from '@material-ui/core/CssBaseline';
+import TextField from '@material-ui/core/TextField';
+import Paper from '@material-ui/core/Paper';
+import Box from '@material-ui/core/Box';
+import Hidden from '@material-ui/core/Hidden';
+import Snackbar from '@material-ui/core/Snackbar';
+import { Link } from 'react-router-dom';
+import Grid from '@material-ui/core/Grid';
+import IconButton from '@material-ui/core/IconButton';
+import CloseIcon from '@material-ui/icons/Close';
+import { Formik } from 'formik';
+import * as Yup from 'yup';
 import Typography from '@material-ui/core/Typography';
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles((theme) => ({
+	root: {
+		minHeight: '100vh',
+		'& .MuiInput-underline:before': {
+			borderBottom: '1.2px solid rgba(0, 0, 0, 0.2)'
+		}
+	},
+	welcome: {
+		fontSize: 26,
```

Comment:
These font sizes should be in the theme based on the variant sizing you are choosing so we are consistent across the whole application